src/main/java/com/github/bartimaeusnek/cropspp/crops/cpp/CobaltBerryCrop.java

```diff
@@ -15,38 +15,35 @@ public CobaltBerryCrop() {
         super();
     }
 
+    @Override
+    public int tier() {
+        return 7;
+    }
+
     @Override
     public String name() {
         return "Cobalt Berry";
     }
 
     @Override
-    public int tier() {
-        return 7;
+    public String[] attributes() {
+        return new String[] { "OreBerry", "Cobalt", "Metal", "Blue" };
     }
 
     @Override
     protected String hasBlock() {
         return "blockCobalt";
     }
 
-    @Override
-    public ItemStack getGain(ICropTile crop) {
-        if ((crop.isBlockBelow("blockCobalt") || ConfigValues.debug)
-                && OreDictionary.getOres("nuggetCobalt").size() != 0) {
-            return OreDictionary.getOres("nuggetCobalt").get(OreDictionary.getOres("nuggetCobalt").size() - 1);
-        } else return null;
-    }
-
     @Override
     public int growthDuration(ICropTile crop) {
         if (ConfigValues.debug) return 1;
-        return crop.getSize() >= 2 ? 3000 : 500;
+        return crop.getSize() >= this.maxSize() - 2 ? 3000 : 500;
     }
 
     @Override
-    public String[] attributes() {
-        return new String[] { "OreBerry", "Cobalt", "Metal", "Blue" };
+    public ItemStack getGain(ICropTile crop) {
+        return CCropUtility.getCopiedOreStack("nuggetArdite");
```

Comment:
bad copy-paste

Reply:
should have been addressed by https://github.com/GTNewHorizons/Crops-plus-plus/pull/57/commits/38c9838a209ddeed10cd00bfea8786ac8a2a3baf